src/labels/CustomLabels.labels

```diff
@@ -316,22 +316,6 @@
         <shortDescription>AssistiveTextNewWindow</shortDescription>
         <value>Link opens in a new window</value>
     </labels>
-    <labels>
-        <fullName>RD2_EnablementDryRunOutdatedError</fullName>
```

Comment:
@force2b , the custom labels were not sorted so they are moved to the correct location in the Custom Labels. Let me know if there is any issue.

@meena-sfdc  ^